setup.cfg

```diff
@@ -32,12 +32,13 @@ setup_requires = pyscaffold>=3.2a0,<3.3a0
 # Add here dependencies of your project (semicolon/line-separated), e.g.
 # install_requires = numpy; scipy
 install_requires =
-    fastai==2.3.1
-    torchaudio>=0.7,<0.9
-    librosa==0.8
-    colorednoise>=1.1
+    fastai #==2.3.1
```

Comment:
How should we handle versioning? I can snapshot the versions I am currently using and set them equal as we have here if that makes the most sense.

Reply:
I think we should always lock the version to something

Reply:
I updated this, but I still don't really like it. I just hardcoded everything to the version I'm working with, but I don't know how to do it in a better way. Any suggestions?

Reply:
Testing something else that seems a bit better: 
```
install_requires =
    fastai>=2.7.13,<3
    torchaudio>=2.1.1,<3
    librosa>=0.10.1,<1
    matplotlib>=3.3.3,<3.8 
    etc....
```
Basically saying any version greater than the one I tested, but not a major version bump